### ISSUE TITLE
WebDriverManager cannot figure out IE version 

### ISSUE THREAD

**Issue by codenikhil1** [bonigarcia/webdrivermanager#830]
**Description of the problem**: WebDriverManager cannot figure out the IE Browser version and compatible driver and end up downloading IEServerDriver 4.0.0.
Because of this execution does not get start and when i manually provide IEServerDriver 3.14 then execution get started.

**Browser and version**: Internet Explorer 21H1

**Operating system**: Windows 10.

**WebDriverManager version**: 5.1.1

**WebDriverManager call**:
 WebDriverManager webDriverManager = WebDriverManager.getInstance(DriverManagerType.IEXPLORER)
 webDriverManager.setup(); 
(same code snippet works well for other browsers)
  **WebDriverManager traces**:Attaching screenshot of WebDriverManager class which i took while debugging the code

![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/51852902/168256032-9a6f8cd8-6d9c-4d70-a05d-1a1d1771431c.png)
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/51852902/168256042-95488ebc-e2a1-421f-bf31-46706b0b043e.png)

**Comment by bonigarcia** [comment]
Indeed, WebDriverManager is not able to discover the IE version. That feature is available for Chrome, Firefox, Edge, Opera, Chromium, and Safari, but not IE.
